examples/friends/mix.exs

```diff
@@ -0,0 +1,36 @@
+defmodule Friends.Mixfile do
+  use Mix.Project
+
+  def project do
+    [app: :friends,
+     version: "0.1.0",
+     elixir: "~> 1.3",
+     build_embedded: Mix.env == :prod,
+     start_permanent: Mix.env == :prod,
+     deps: deps()]
+  end
+
+  # Configuration for the OTP application
+  #
+  # Type "mix help compile.app" for more information
+  def application do
+    [applications: [:logger, :postgrex],
+     mod: {Friends, []}]
+  end
+
+  # Dependencies can be Hex packages:
+  #
+  #   {:mydep, "~> 0.3.0"}
+  #
+  # Or git/path repositories:
+  #
+  #   {:mydep, git: "https://github.com/elixir-lang/mydep.git", tag: "0.1.0"}
+  #
+  # Type "mix help deps" for more examples and options
+  defp deps do
+    [
+      {:ecto, "2.0.1"},
+      {:postgrex, "0.11.2"}
```

Comment:
Let's use `path` dependency for ecto with a comment on how to use this in a proper app, similar to the other example: https://github.com/elixir-ecto/ecto/blob/master/examples/simple/mix.exs#L16-L19